cmd/empire/main.go

```diff
@@ -98,6 +106,12 @@ var Commands = []cli.Command{
 				EnvVar: "EMPIRE_SCHEDULER",
 			},
 			cli.StringFlag{
+				Name:   FlagSAMLMetadata,
+				Value:  "",
+				Usage:  "The location of the SAML metadata XML (e.g. https://app.onelogin.com/saml/metadata/1234)",
+				EnvVar: "EMPIRE_SAML_METADATA",
```

Comment:
Right. This can take the raw content, a url, or a file path to the content. I'll update the flag usage to reflect that.